src/platform/graphics/null/null-graphics-device.js

```diff
@@ -17,6 +18,14 @@ class NullGraphicsDevice extends GraphicsDevice {
         this.isNull = true;
         this._deviceType = DEVICETYPE_NULL;
         this.samples = 1;
+
+        this.backBuffer = new RenderTarget({
+            name: 'Framebuffer',
```

Comment:
maybe call it null-framebuffer or similar

Reply:
had that but seemed strange ;)
anyways, I don't think this matters on Null renderer, nobody captures / debugs this as it does nothing